### ISSUE TITLE
Indentation of link and script is not working properly

### ISSUE THREAD

**Issue by AirBorne04** [negibouze/html-webpack-pug-plugin#14]
Hi,

first of all THANK YOU for this package this is really needed and very useful when usig server side pug templates!!

I came across a little issue on the way the indentation is working on bundled js and css files. In the code I identified the following issue. [hlSpace](https://github.com/negibouze/html-webpack-pug-plugin/blob/f3f3f07adc2bf743263d3e4093778dff6f42a579/index.js#L335) seems to be doubling the space from the body (usually e.g. 2 spaces) element. This is fine if you are going with a normal template like
```
html
  head
  body
```

In my case my template looks like
```
if true
  html
    head
      link(id="myLink")
    body
```

Doubling the space of the body (here 4 spaces) will create one indentation to much on the result file and ends up like this:
```
if true
  html
    head
      link(id="myLink")
        link(href="bundle.css")
    body
```

So that the `bundle.css` is indented to far and makes `bundle.css` a child node of `myLink`.

**Comment by negibouze** [comment]
@AirBorne04 
Thank you for you report.
I will try to improve.

**Comment by negibouze** [comment]
@AirBorne04 

I released a new version(v0.2.2).
Please try it.

**Comment by AirBorne04** [comment]
This is fixing my issue, thank you for your quick action :)
@negibouze
